docs/conf.py

```diff
@@ -203,7 +203,16 @@ def _load_theme(
 # Theme options are theme-specific and customize the look and feel of a theme
 # further.  For a list of options available for each theme, see the
 # documentation.
-# html_theme_options = {}
+
+# Dark by default
+html_theme_options = {
+    "light_css_variables": {
+        "color-scheme": "dark",
+        },
+    "dark_css_variables": {
+        "color-scheme": "dark",
+        },
+    }
```

Comment:
For me `furo` already automatically enables dark mode for example on: https://scikit-build-core.readthedocs.io/en/latest/. I have it enabled system wide on the desktop (and firefox by proxy I guess)

Reply:
From the documentation, it's light by default, but switches to dark automatically when browser is set to prefer dark.

Reply:
Welp, no reason to complicate things. When I've had an epaper device, the dark-theme sites were unreadable, so in some edge cases it's better to let the system/browser set what it prefers, and let the users set it up or buy a good pair of sunglasses.

Reply:
Yep, agree, I'm just playing with it while in Draft. Also not happy with introducing js :)